src/PharoThings-Hardware-RaspberryPi.package/RpiBoard3B.class/instance/pin27.st

```diff
@@ -0,0 +1,3 @@
+initialization
+pin27
+	^ PotSDAPin id: 0
```

Comment:
What about the logic for names?
Why it is pin27 but id is 0? Or in previous method pin13 but id is 27?

Reply:
I'm using the physical position to name the methods because they will never change. 
And the id: is the BCM number and number: is the WiringPi number. 
I think was easier to understand if they were named bcmNumber: and wpiNumber. And maybe renew pin27 to phyPin27 or physicalPin27. What do you think?

![image](https://user-images.githubusercontent.com/39618015/58085224-9ecfd700-7bbc-11e9-922c-da3a6f121e0c.png)

Reply:
I think we should move to this approach, using the physical Pins to configure the devices.
To configure the LCD driver now, for example, we need to know the number of BCM. 
Using a physical position this process becomes more transparent, it's just seen the Pin position on the physical board. 

`modePin := aBoard physicalPin13.
clockPin :=  aBoard physicalPin15.`

Reply:
> And the id: is the BCM number and number: is the WiringPi number.
> I think could be easier to understand if they were named **bcmNumber:** and **wpiNumber:**. And maybe renew pin27 to phyPin27 or **physicalPin27**. What do you think?

#number is not wpiNumber. Maybe they are equal in most of cases but it was not intentional.
I added number only for informative purpose to show gpio pins with same name as they are shown in pinout schemas. So the wiringPi id is not used at all.

There is already methods to get physical position of pins:
```Smalltalk
gpio2 position "==> 1@7"
gpio2 headerNumber "==> 13"
``` 
Position is a variable and headerNumber is computed from it.
Position is set to the pin during initialization of pin scheme. We define scheme in array and position is simply computed from the index in this array.

Reply:
> Will method #pinAtHeader: covers this idea?
> (I don't think that we need a method for every pin)

I removed the methods to every pin and create a dictionary with them. I think #pinAtHeader: cannot be used because in this part, because the pins was not setted yet.

Reply:
> But anyway it is up to user to decide how to get pins from the board.
> We just need to provide more suitable methods to access pins:

I add connector: in some methods to support select the connector.